ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -231,28 +236,38 @@ export const Homepage = withUserProfile()(class extends React.Component<Props, S
     });
   }
 
-  openVideo(videoLink: string): void {
-    this.setState({videoOpen: true, videoLink: videoLink});
+  openVideo(videoId: string): void {
+    this.setState({videoOpen: true, videoId: videoId});
   }
 
 
   render() {
     const {betaAccessGranted, videoOpen, accessTasksLoaded, accessTasksRemaining,
       eraCommonsError, eraCommonsLinked, eraCommonsLoading, firstVisitTraining,
-      trainingCompleted, quickTour, videoLink, twoFactorAuthCompleted,
-      dataUserCodeOfConductCompleted
+      trainingCompleted, quickTour, videoId, twoFactorAuthCompleted,
+      dataUserCodeOfConductCompleted, quickTourResourceOffset
     } = this.state;
+    const limit = this.quickTourResourcesDiv ? Math.floor(this.quickTourResourcesDiv.offsetWidth * 0.97 / 276) : 5;
 
     const quickTourResources = [
       {
         src: '/assets/images/QT-thumbnail.svg',
         onClick: () => this.setState({quickTour: true})
       }, {
-        src: '/assets/images/cohorts-thumbnail.png',
-        onClick: () => this.openVideo('/assets/videos/Workbench Tutorial - Cohorts.mp4')
+        src: '/assets/images/intro-workbench.png',
+        onClick: () => this.openVideo('nqnOvOpnRLE')
```

Comment:
Out of curiosity, why are the video titles hard-coded with random characters? I wonder if there's a more straightforward way to store this mapping, outside of code (like in an environment config file).

Reply:
Those are the YouTube video ids that get inserted into the src url of the iframe. Yeah, not sure the best way to do it but didn't want to change too much since I'm not usually in these components.

Reply:
I would personally set the id to a constant above, and then put that exact comment about YouTube video ids above it (literally I'd just copy word for word)

Reply:
Added a comment above. Not sure what you mean about setting the id to a constant.